client.go

```diff
@@ -312,6 +312,11 @@ type RemoteContext struct {
 	// afterwards. Unpacking is required to run an image.
 	Unpack bool
 
+	// Deduplicate is a boolean flag to specify whether to allow GC to clean
+	// layers up from the content store after successfully unpacking these
+	// contents to the snapshotter.
+	Deduplicate bool
```

Comment:
I don't think "deduplication" is the right term here. I think it is about "preservation" of the content. The content is preserved to allow redistributing the artifact or future unpacking. The content and the unpacked snapshot are not duplicates as they server different purposes. For example, it may be that different snapshotters may be desired for different use cases or runtimes, having the ability to unpack again may desired. However you cannot just copy over the other snapshot and expect to get the exact same content.

Reply:
Thanks for the review. I changed the terms to use "preservation".